IntercomDotnet/Resources/Users.cs

```diff
@@ -26,7 +26,7 @@ public dynamic Delete(object hash)
                 });
         }
 
-        public dynamic Get(string email = null, int? userId = null)
+        public dynamic Get(string email = null, string userId = null)
```

Comment:
Is it worth providing this as an overload? That would stop any breakages for existing users if they update the library. The old function (this one) can be marked as ```[Obsolete]``` to get users to move over in time?